content/docs/context.md

```diff
@@ -57,18 +57,17 @@ It might feel redundant to pass down the `user` and `avatarSize` props through m
 One way to solve this issue **without context** is to [pass down the `Avatar` component itself](/docs/composition-vs-inheritance.html#containment) so that the intermediate components don't need to know about the `user` prop:
 
 ```js
-function Page(props) {
-  const user = props.user;
+function Page({ user, avatarSize }) {
```

Comment:
None of the other function components on the page use destructuring for props, can we keep this the same and just add the missing prop below?

Reply:
@alexkrolick Thanks for you reply. Sure, I'll update this PR to match the standards.

Reply:
@alexkrolick seems that it was fixed 23 hour ago 👍 https://github.com/reactjs/reactjs.org/commit/8e805d799501b2cc7db3a9726272f90228fc1916

Reply:
Closing this PR